README.md

```diff
@@ -15,39 +15,43 @@ The bootstrap workflow uses the following process to handle incoming EDI:
 The bootstrap uses a similar process in reverse to handle outgoing EDI. It translates JSON documents to EDI and publishes them to a Stedi bucket.
 
 
-## Prerequisites & Deployment
+## Requirements
 
-1. [Node.js](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) _(minimum version: 15)_
+1. Install [Node.js](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) _(minimum version: 15)_
 
-1. Clone this repo and install the necessary dependencies:
+1. Clone the bootstrap repository and install the necessary dependencies:
 
    ```bash
    git clone https://github.com/Stedi-Demos/bootstrap.git
    cd bootstrap
    npm ci
    ```
 
-1. Go to [webhook.site](https://webhook.site/) and copy the unique URL. The demo will send output to this webhook.
+1. Create a [Stedi account](https://www.stedi.com/auth/sign-up?) and [generate an API Key](https://www.stedi.com/app/settings/api-keys). An API key is required to authenticate with Stedi APIs and deploy Stedi functions. 
```

Comment:
I agree with the addition, but I would argue that creating an account and generating an API key are two separate steps. As long as we’re picking nits. 😉

Reply:
```suggestion
1. Create a [Stedi account](https://www.stedi.com/auth/sign-up?).

 1. [Generate an API Key](https://www.stedi.com/app/settings/api-keys). An API key is required to authenticate with Stedi APIs and deploy Stedi functions. 
```